lib/facter/pe_status_check_role.rb

```diff
@@ -0,0 +1,36 @@
+Facter.add(:pe_status_check_role) do
+  require 'puppet'
+  require_relative '../shared/pe_status_check'
+
```

Comment:
needs  confine { Facter.value(:pe_build) } otherwise this will run on every node